test/system/rich_text_editor_test.rb

```diff
@@ -27,6 +31,26 @@ def setup
 
     # Make sure that image has been uploaded
     assert_equal( is_image_url_undefined, false )
+
+    find('.ple-publish').click()
+
+    assert_selector('h1', text: "My note")
+    assert_selector('#content', text: "All about this interesting stuff")
+    assert_selector('.alert-success', text: "×\nResearch note published. Get the word out on the discussion lists!")
+
+    find('.main-image img.d-print-none')
```

Comment:
let's try this and the line below see what happens...

Reply:
For nested selectors we could try something like:

https://github.com/publiclab/plots2/blob/d2a89b0a22289341431faa09e8a8f355e44a0cef/test/system/spam2_test.rb#L155-L157

Reply:
```suggestion
    within ".main-image" do
      expect(find('img.d-print-none'))
    end
```

Or we could try the format: 

```
    assert_select "img.main-image-tag" do
      assert_select "[src=?]", /pl.png/
    end
```